src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1658,7 +1735,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
    -->
    <ItemGroup>
       <_MSBuildProjectReferenceExistent Condition="'%(_MSBuildProjectReferenceExistent.SkipGetTargetFrameworkProperties)' == '' and ('%(Extension)' == '.vcxproj' or '%(Extension)' == '.nativeproj')">
-        <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
+        <!-- When we're dynamically figuring out platform, we need the MSBuild call that retrieves TF data. -->
```

Comment:
Are you suggesting changing the comment? But yes, because we're piggybacking off of `GetTargetFrameworks`, we explicitly need that to run when opted into this feature.

Reply:
I guess the question is, should this silently change user configuration or should it emit an error?

Reply:
I think it's safe to say that we're not "silently" changing configuration because this is opt in, but we should make it super explicit in the documentation that an extra target runs for all cpp projects.

Reply:
Ah, you're right since this is just avoiding the code we put in earlier to keep C++ from paying the negotiation cost.